src/main/java/com/switchfully/parkshark_2024_10/division/DivisionService.java

```diff
@@ -0,0 +1,24 @@
+package com.switchfully.parkshark_2024_10.division;
+
+import com.switchfully.parkshark_2024_10.division.dto.CreateDivisionDto;
+import com.switchfully.parkshark_2024_10.division.dto.DivisionDto;
+import jakarta.transaction.Transactional;
+import org.springframework.stereotype.Service;
+
+@Service
+@Transactional
+public class DivisionService {
+    DivisionRepository divisionRepository;
+    DivisionMapper divisionMapper = new DivisionMapper();
```

Comment:
Not sure why you are initializing the DivisionMapper here, as it is probably only used as a Utility Class, which only contains static methods to map from one Class to another.